core/primitives/version.hpp

```diff
@@ -60,6 +68,19 @@ namespace kagome::primitives {
     /// List of supported API "features" along with their versions.
     ApisVec apis;
   };
+
+  /**
+   * @brief outputs object of type Version to stream
+   * @tparam Stream stream type
+   * @param s stream reference
+   * @param v value to output
+   * @return reference to stream
+   */
+  template <class Stream>
+  Stream &operator<<(Stream &s, const Version &v) {
+    return s << std::string_view(v.spec_name) << std::string_view(v.impl_name)
```

Comment:
why string view?

Reply:
I implemented encoding string_view as one of base types in ScaleEncoderStream, and spec_name is a string. That's why.

Reply:
string is convertible to string_view. You can safely remove cast to string_view